components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Twin/tests/Control/AddressSpaceBrowseTests.cs

```diff
@@ -18,19 +22,26 @@ public class AddressSpaceBrowseTests {
 
         public AddressSpaceBrowseTests(TestServerFixture server) {
             _server = server;
+            _hostEntry = Try.Op(() => Dns.GetHostEntry(Utils.GetHostName()))
+                ?? Try.Op(() => Dns.GetHostEntry("localhost"));
         }
 
         private BrowseServicesTests<EndpointModel> GetTests() {
+
             return new BrowseServicesTests<EndpointModel>(
                 () => new AddressSpaceServices(_server.Client,
                     new VariantEncoderFactory(), _server.Logger),
                 new EndpointModel {
-                    Url = $"opc.tcp://{Dns.GetHostName()}:{_server.Port}/UA/SampleServer",
+                    Url = $"opc.tcp://{_hostEntry?.HostName}:{_server.Port}/UA/SampleServer",
```

Comment:
If `_hostEntry` is null, then URL will have `opc.tcp://:51510/UA/SampleServer` value instead of `null`, which does not look like a correct value.

Reply:
well, if the hostEntry fails to resolve "localhost" then let it just crash the test...

Reply:
maybe you can do that in the constructor, throw when second case also fails ?

Reply:
yeah, probably I should fallback to localhost if the hostEntity resolution failed

Reply:
done